server.js

```diff
@@ -1,35 +1,31 @@
 // TASK 1: Setting up server
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');

```

Comment:
Consider removing the redeclaration of the 'PORT' variable. Since it is already defined on line 2, there is no need to redefine it on line 9.